tests/gcoap_fileserver/main.c

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2022 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       test application for the GCoAP file server
+ *
+ * @author      Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ * @}
+ */
+
+#include "fs/constfs.h"
+#include "net/gcoap.h"
+#include "net/gcoap/fileserver.h"
+#include "shell.h"
+#include "vfs_default.h"
+
+#define MAIN_QUEUE_SIZE (4)
+static msg_t _main_msg_queue[MAIN_QUEUE_SIZE];
+
+static const gcoap_fileserver_entry_t _vfs_entry = {
+    .root = "/const",
+    .resource = "/vfs",
```

Comment:
This should match whatever it's mounted under -- either it's `/vfs` below or also `/const` here.

(It doesn't bite in the tests because we don't go through directory listings)

Reply:
let's hope this soon becomes obsolete with #17957